pack.yaml

```
@@ -8,7 +8,7 @@ keywords:
    - ncm
    - npm
    - monitoring
-version: 0.7.10
+version: 0.8.4
```

Is the next version not 0.8.0?

```suggestion
version: 0.8.0
```

---

I still have lots to learn about versioning.  I had bumped it to 0.8.0 initially but then I went back and added 2 more actions to my repo so I bumped the Rev from 0 to 4 to reflect those changes.  I have dropped it back to 0.8.0